### DISCUSSION TITLE
blog/2022/220414-fk-arcaea-songlist-hash/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#5] [Announcements]
# blog/2022/220414-fk-arcaea-songlist-hash/

Arisa 的个人博客

https://blog.arisa.moe/blog/2022/220414-fk-arcaea-songlist-hash/

**Comment by Kry-ptvn** [comment]
这个方法适用于3.12.6以上的版本吗

**Reply by Kry-ptvn** [reply]
我试过,大陆官网c版是不适用的

**Reply by Kry-ptvn** [reply]
还有armeabi v7a

**Reply by zhanbao2000** [reply]
616 长期以来都没有修改过他们的检测方式。我手上的最新版本是 4.1.1 的 64 位版本，至少这个版本是能用的，armv7 我没有去试

**Reply by Kry-ptvn** [reply]
还有个问题想请教一下:

"最后这里是直接调用 exit() 函数。那看来就是这了。
回到 IDA View，看看 exit() 函数在哪。
......
找到调用 exit() 函数的标签 loc_9459F0，向上高亮所有跳转到这个标签的指令。"

是按Tab键跳到对应的IDA-view窗口吗?

**Reply by Kry-ptvn** [reply]
4.1.4也成功了(好像

**Reply by dydydydyd** [reply]
请问这个方法4.1.9c版本的能用吗，好像没有找到exit函数
